infinity-app/app.R

```diff
@@ -118,7 +161,7 @@ ui <-
         inputId = "taxa_genus",
         label = "Choose a genus: (you can also select All, but it's slow so be patient)",
```

Comment:
Is it still slow to choose 'All'? and if so, should it really be the default?

Reply:
good point, changing